### ISSUE TITLE
Question about antlr4 dependency & minimum Node version

### ISSUE THREAD

**Issue by cgewecke** [solidity-parser/parser#33]
(Thanks a million for maintaining this @fvictorio. It's the best.)

In 71f83ca96f116977011770d6761f54316e38652b antlr4 is added as a direct dependency. 

Was updating some things to `0.9.1` and am getting the installation error below when is Node < 14. 

14 is LTS as of late October but just wanted to double-check with you that this is expected. 

```
[2/4] 🚚  Fetching packages...
error antlr4@4.9.0: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.16.0"
error Found incompatible module.
```

**Comment by cgewecke** [comment]
Sorry, I just found the relevant antlr4 docs and they say that's their target. 😅

**Comment by fvictorio** [comment]
But you are rights, this shouldn't happen. I'm planning on packaging this library with rollup or something like that, and to remove `antlr4` as a dependency. That way this library will continue to work with any version.

**Comment by fvictorio** [comment]
Fixed on `@solidity-parser/parser@0.10.1`
